### PR TITLE
Database From Diagram

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer,

In this PR we did the following tasks:

- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Create the relationship between the Animals tables.
- Added visual representation with the PK/FK notation.

Happy Review!

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    name VARCHAR(100),
+    description TEXT
+);
+
+-- Assuming a many-to-many relationship between Patients and Medications
+
+CREATE TABLE Patient_Medications (
+    patient_id INT,
+    medication_id INT,
+    start_date DATE,
```
Comment (mikethreels) [inline schema_based_on_diagram.sql]: - [ ] the schema you created here is not based on the provided diagram, the table names are not corresponding and also the columns in the tables are not the same, please make sure the schema is created based on the provided diagram.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job writing the queries.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE specializations(
```
Comment (V-Blaze) [inline schema_based_on_diagram.sql]: - You have done great so far but there are still some project requirements you need to meet :+1: 

- DATABASE-DIAGRAM:
Screenshot from 2023-09-05 22-50-14


- [ ] Just as the previous reviewer mentioned, you are meant to follow the diagram above and create all the tables and columns in it. You are meant to create the `medical_histories, treatments, invoice, invoice_items, patients` table and all the columns in them. and also implement all the relationships both by using foreign key and join tables. I'lll suggest you go through the project requirement carefully and implement this. :smile: :wave:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @MasumaJaffery, @cancelei, @Grandi0z    

# Almost there 🟡 

1094a3e620ebea4cd6cf2d0aceaa8f52


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
-  Correct Git flow✔
- Readme looks professional 💯 🥇 ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (codepantha) [reply schema_based_on_diagram.sql]: image
- [ ] Just like the previous reviewer has pointed out, please follow the attached diagram above ☝️ and create all the tables and their respective columns. Please see the project requirements below 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow 📌 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descriptive PR⭐
- [ ]  Kudos for working on functionality for the schem🤩
- [ ]  No Linters error ✅



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**